fs/mmap/fs_mmap.c

```diff
@@ -272,11 +252,11 @@ int file_mmap(FAR struct file *filep, FAR void *start, size_t length,
 FAR void *mmap(FAR void *start, size_t length, int prot, int flags,
                int fd, off_t offset)
 {
-  FAR struct file *filep;
+  FAR struct file *filep = NULL;
```

Comment:
Initializing here is not needed, since fs_getfilep initializes filep unless it fails, in which case filep is not used.